.gitpod.Dockerfile

```diff
@@ -1,10 +1,15 @@
-FROM gitpod/workspace-full
-
-# Python Dependencies
-ARG PYTHON_VERSION=3.9
-
-RUN sudo add-apt-repository ppa:deadsnakes/ppa -y
-RUN sudo apt install python${PYTHON_VERSION} -y
-RUN sudo apt install python${PYTHON_VERSION}-venv -y
-RUN sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python${PYTHON_VERSION} 2
-RUN sudo pip install --upgrade pip
+FROM gitpod/workspace-python
+
+RUN pyenv install 3.9.13 \
+    && pyenv global 3.9.13
+
+# FROM gitpod/workspace-full
+
+# Python Dependencies
+# ARG PYTHON_VERSION=3.9.13
+
+# RUN sudo add-apt-repository ppa:deadsnakes/ppa -y
+# RUN sudo apt install python${PYTHON_VERSION} -y
+# RUN sudo apt install python${PYTHON_VERSION}-venv -y
+# RUN sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python${PYTHON_VERSION} 2
+# RUN sudo pip install --upgrade pip
```

Comment:
Should just remove this for now. We have the legacy to reference for sorting this out.